src/FlipMove.js

```diff
@@ -22,70 +27,40 @@ import {
   whichTransitionEvent,
 } from './dom-manipulation';
 import { arraysEqual } from './helpers';
+import type {
+  ConvertedProps,
+  FlipMoveState,
+  ElementShape,
+  ChildrenHook,
+  ChildData,
+  NodeData,
+  DelegatedProps,
+  Styles,
+} from './typings';
 
 const transitionEnd = whichTransitionEvent();
 const noBrowserSupport = !transitionEnd;
 
+function getKey(childData: ChildData): string {
+  return childData.key || '';
+}
```

Comment:
This was more helpful when it was `childData.element.key || ''`, but I decided to keep it, because when the shape of `ChildData` changes again, it will be a matter of changing the helpe's body instead of ten replacements